doc: Drop read only functionality for bl_storage library from app (#9418)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Bootloader libraries
--------------------

>   * :ref:`doc_bl_storage`: Removed option to use the library to read out OTP memory from the application, when the |NSIB| (NSIB) is used.

Reviewer: ```suggestion
  * :ref:`doc_bl_storage`:
  
    * Removed the option of application using the library to read OTP memory when the |NSIB| (NSIB) is enabled.
```
